gravitee-apim-perf/src/scenarios/message-kafka/http-post/keyless.test.ts

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { check } from 'k6';
+import { Connection } from 'k6/x/kafka';
+import http from 'k6/http';
+import { ADMIN_USER, authorizationHeaderFor, k6Options } from '@env/environment';
+import { LifecycleAction } from '@models/v3/ApiEntity';
+import { failIf } from '@helpers/k6.helper';
+import { ApisFixture } from '@fixtures/v3/apis.fixture';
+import { HttpHelper } from '@helpers/http.helper';
+import { GatewayTestData } from '@lib/test-api';
+import { GatewayClient, HttpMethod } from '@clients/GatewayClient';
+import { ApisV4Fixture } from '@fixtures/v4/apis.v4.fixture';
+import { ApisV4Client } from '@clients/v4/ApisV4Client';
+import { PlansV4Client } from '@clients/v4/PlansV4Client';
+import { PlansV4Fixture } from '@fixtures/v4/plans.v4.fixture';
+import { NewPlanEntityV4StatusEnum } from '@models/v4/NewPlanEntityV4';
+import { ApiEntityV4 } from '@models/v4/ApiEntityV4';
+import { PlanEntityV4 } from '@models/v4/PlanEntityV4';
+import { NewApiEntityV4TypeEnum } from '@models/v4/NewApiEntityV4';
+
+export const options = k6Options;
+
+const connection = new Connection({
+  // ConnectionConfig object
+  address: k6Options.apim.kafkaBoostrapServer,
+});
+
+const kafkaTopic = k6Options.apim.httpPost.topic;
+const numPartitions = k6Options.apim.httpPost.numPartitions;
+
+
+if (__VU == 0) {
```

Comment:
Is it a variable coming from K6?

Reply:
yes, according to the documentation it is discouraged to use it but all the examples of the Kaka extension are using this global variable before doing a createTopic or a deleteTopic. So I did the same to avoid issues.